### DISCUSSION TITLE
3.6.2 client installers?

### DISCUSSION THREAD

**Discussion by chrisrimple** [jamulussoftware/jamulus#1499] [Technical Support & Help]
My group is having significant issues with scratchiness and sound disruption.  We're all running 3.7.0 on a 3.7.0 server, most using Macs.  I want to downgrade everything to 3.6.2.  Are the legacy compiled client installers still available somewhere?

**Comment by hoffie** [comment]
This is sad to hear. There have been some reports about differences with 3.7.0, but I think all of them have turned out to be unrelated to the update. Please let us know if you find otherwise!

The installers for previous versions should be here: https://sourceforge.net/projects/llcon/files/Jamulus/3.6.2/
